spec/system/group_spec.rb

```diff
@@ -67,4 +63,66 @@
 
     expect(page).to have_text("Group was successfully updated.")
   end
+
+  it "deletes the group" do
+    visit user_groups_path(@user)
+    click_on "Delete"
+    button = find(id: "groups-group-delete-button")
+    button.click
+
+    expect(page).to have_text("Group was successfully deleted.")
+  end
+
+  it "add secondary mentor" do
+    visit "/groups/#{@group.id}"
+    click_button "+ Add Mentors"
+    fill_in_input "#group_email_input_mentor", with: @user3.email
+    fill_in_input "#group_email_input_mentor", with: :enter
+    click_button "Add mentors"
+
+    expect(page).to have_text(
+      "Out of 1 Email(s), 1 was valid and 0 were invalid. 1 user(s) will be invited."
+    )
+  end
+
+  it "remove secondary mentor" do
+    GroupMember.create(group_id: @group.id, user_id: @user3.id, mentor: true)
+    visit "/groups/#{@group.id}"
+    click_on "Remove"
+    execute_script "document.getElementById('deletememberModal').style.display='block'"
+    execute_script "document.getElementById('deletememberModal').style.opacity=1"
+    click_on "Delete"
+
+    expect(page).to have_text("Group member was successfully removed.")
+  end
+
+  it "convert member to mentor" do
+    GroupMember.create(group_id: @group.id, user_id: @user2.id, mentor: false)
```

Comment:
Can we move this line to a `let` statement to avoid code repetition?

Reply:
yes we can do that